internal/server/routes.go

```diff
@@ -123,6 +123,14 @@ func (s *Server) GenerateRoutes() Routes {
 	get(a, noAuthnWithOrg, "/api/providers/:id", a.GetProvider)
 	get(a, noAuthnWithOrg, "/api/providers", a.ListProviders)
 	get(a, noAuthnWithOrg, "/api/settings", a.GetSettings)
+	add(a, noAuthnWithOrg, route[api.VerifyAndRedirectRequest, *api.RedirectResponse]{
```

Comment:
I think this will conflict with the changes in #3175 (just merged). With those changes you can move the `route` out to a package-level var, so that only the group, method, and path are inline here.